carbonmark-api/src/utils/helpers/utils.ts

```diff
@@ -47,7 +47,11 @@ export async function getAllVintages(
   digitalCarbonProjects.forEach((project) => {
     project.carbonCredits.forEach((credit) => {
       if (credit.vintage) {
-        uniqueValues.add(credit.vintage.toString());
+        if (
+          credit.poolBalances?.some((balance) => Number(balance.balance) > 0)
+        ) {
+          uniqueValues.add(credit.vintage.toString());
+        }
```

Comment:
Just a note that this will filter out all the ICR vintages as there are no pool balances for those, just direct seller to buyer listings. 

This isn't currently an issue as the ICR filters are fetched separately from their API at the moment, but the eventual goal is get everything from CM subgraphs and not outside sources.

Reply:
@psparacino this project is not part of a pool and does show up when filtering https://www.carbonmark.com/projects/VCS-1764-2020

also note we are in the process of adding a "contains seller listing" toggle to the filtering tools
https://github.com/KlimaDAO/klimadao/issues/1054

Reply:
Oh, I just meant the vintage from the project not the project itself. Given that there is so much overlap in the common vintages it's probably a non-issue